modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -153,15 +188,21 @@ $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_uti
 
 . Once all the deployments for the cluster have been rolled out, re-enable shard balancing:
 +
+[source,terminal]
 ----
 $ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util --query=_cluster/settings -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "transient": { "cluster.routing.allocation.enable" : "none" } }'
 ----
 +
 For example:
 +
+[source,terminal]
 ----
 $ oc exec elasticsearch-cdm-5ceex6ts-1-dcd6c4c7c-jpw6 -c elasticsearch -- es_util --query=_cluster/settings?pretty=true -XPUT 'https://localhost:9200/_cluster/settings' -d '{ "transient": { "cluster.routing.allocation.enable" : "all" } }'
-
+----
++
+Example output
+[source,terminal]
```

Comment:
I've seen source,yaml used for commands that produce only YAML, so it might make sense to use source,json for this because it outputs JSON.